test/integration/wp-env-schema.test.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import Ajv from 'ajv';
+import glob from 'fast-glob';
+
+/**
+ * Internal dependencies
+ */
+import wpEnvSchema from '../../schemas/json/wp-env.json';
+
+describe( '.wp-env.json schema', () => {
+	const jsonFiles = glob.sync( [ '.wp-env.json' ], { onlyFiles: true } );
```

Comment:
I don't think any new `.wp-env.json` files will be added in the future, but I am concerned that there may be a better pattern.

Reply:
I'd remove the `jsonFiles` array entirely if there's only one.

- Import the `.wp-env.json` file just like the schema as an internal dependency.
- Remove the `'found .wp-env.json files'` test.
- Replace the `test.each` with a singular test.
- Replace the `require( filepath )` with the imported variable.